plugins/catalog-backend-module-msgraph/src/processors/MicrosoftGraphOrgReaderProcessor.ts

```diff
@@ -33,9 +33,10 @@ import {
 } from '../microsoftGraph';
 
 /**
- * Extracts teams and users out of a the Microsoft Graph API.
+ * Extracts teams and users out of the Microsoft Graph API.
  *
  * @public
+ * @deprecated Use the MicrosoftGraphOrgEntityProvider instead.
```

Comment:
likely a bit controversial I guess and related to the general question I have about such processors.

Reply:
This make sense given that the entity provider archives the same result. For all of the deprecations I think we should include some `log.warn` pointing to docs on how to actually upgrade so that we give some heads up and a (hopefully) smoother migration :)

Reply:
Yes, agree. I planned to do that still. Pointing to docs is a good point.